packages/runner/src/app/app.scss

```diff
@@ -28,6 +28,22 @@
   border-right: solid 1px #c7c7c7;
 }
 
+// if you change this here,
+// make sure to update this in app.jsx too
+$spec-list-width: 250px;
```

Comment:
This is the only CSS I added and it's NOT rendered in the app (behind feature flag). I have no idea why Percy is complaining a bunch of stuff is off by 1px.